ur_robot_driver/resources/ros_control.urscript

```diff
@@ -99,7 +99,7 @@ control_mode = MODE_UNINITIALIZED
 thread_move = 0
 global keepalive = -2
 textmsg("ExternalControl: External control active")
-params_mult = socket_read_binary_integer(1+6+1, "reverse_socket")
+params_mult = socket_read_binary_integer(1+6+1, "reverse_socket", 0)
```

Comment:
Those lines could be swapped so that the output only is printed if there was data received from the remote host.

Reply:
addressed in 21a4146ab8cb83dd8bd54e0abee670ee10e70428